t/t2800-jobs-cmd.t

```diff
@@ -639,6 +639,26 @@ test_expect_success 'flux-jobs --format={status},{status_abbrev} works' '
 	test $count -eq $(state_count completed)
 '
 
+test_expect_success 'flux-jobs --format={waitstatus},{returncode}' '
+	FORMAT="{waitstatus:h},{returncode:h}" &&
+	flux jobs --filter=pending,running -no "$FORMAT" > returncodePR.out &&
+	flux jobs --filter=inactive -no "$FORMAT" > returncodeI.out &&
+	test_debug "echo active:; cat returncodePR.out" &&
+	test_debug "echo inactive:; cat returncodeI.out" &&
+	countPR=$(grep -c "^-,-$" returncodePR.out) &&
+	test_debug "echo active got $countPR, want $(state_count sched run)" &&
+	test $countPR -eq $(state_count sched run) &&
+	count=$(grep -c "^32512,127$" returncodeI.out) &&
```

Comment:
nit: should use different filename than `returncodeI.out` for each case

Reply:
I don't follow. Are you suggesting separate test for `{returncode}` vs `{waitstatus}`, or separating failed completed and cancelled job listing into separate output files? (That would seem to unnecessarily slow down this test, which is already one of the longest in the testsuite)

Or am I missing something obvious?

Reply:
oh sorry, my bad ... thought you were outputting to `returncodeI.out` multiple times, but you are actually grepping the same output in `returncodeI.out` multiple times.

So everythings fine :-)